packages/astro/test/astro-global.test.js

```diff
@@ -21,12 +21,12 @@ Global('Astro.request.canonicalURL', async (context) => {
     '/': 'https://mysite.dev/',
     '/post/post': 'https://mysite.dev/post/post/',
     '/posts': 'https://mysite.dev/posts/',
-    '/posts/1': 'https://mysite.dev/posts/', // should be the same as /posts
     '/posts/2': 'https://mysite.dev/posts/2/',
   };
 
   for (const [url, canonicalURL] of Object.entries(canonicalURLs)) {
     const result = await context.runtime.load(url);
+    console.error(result);
```

Comment:
Was this just temporary debug logging?

Reply:
ah, thanks for finding! will remove

Reply:
Somewhere in this PR is a line that does `console.log(meta.glob)` or something... I'll have to find it to ask about it.